command/kv_helpers.go

```diff
@@ -75,7 +75,8 @@ func kvPreflightVersionRequest(client *api.Client, path string) (string, int, er
 				err = fmt.Errorf(
 					`This output flag requires the success of a preflight request 
 to determine the version of a KV secrets engine. Please 
-re-run this command with a token with read access to %s`, path)
+re-run this command with a token with read access to %s. 
+Note if this is a V2 path make sure that you have included data/ to your path.`, path)
```

Comment:
Two potential issues with this error message:

1. People reading this might think that they need to put data/ in the command they ran this flag with, but really they need to include it in the _policy_ used to run this command (at least that's where I believe the issue-opener was running into confusion, based on my own attempts to repro).
2. Vault KV newbies using this flag might not know specifically where data/ goes; they might mistakenly think you mean put it in the very beginning of the path string like `data/secret/foo`.

Might I suggest the following, to spell it out for them even more obviously:

`"Note that if the path you are trying to reach is a KV v2 path, your token's policy must allow read access to that path in the format 'mount-path/data/foo', not just 'mount-path/foo'."`

Reply:
thank you I wasn't sure what the correct wording should be for this error message.